.github/workflows/CI.yaml

```diff
@@ -54,12 +81,13 @@ jobs:
         run: ./gradlew :dataland-frontend:generateAPIClientFrontend :dataland-frontend:npm_run_lintci :dataland-frontend:npm_run_testcomponent :dataland-frontend:npm_run_checkdependencies --no-daemon --stacktrace
       - name: Upload FE lcov Exec Results
         if: always()
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v3
         with:
           name: lcovResults
           path: ./dataland-frontend/coverage/lcov.info
       - name: Execute Frontend Scripts to check if they run successfully
         run: ./gradlew :dataland-frontend:npm_run_fakefixtures --stacktrace
+      - uses: ./.github/workflows/dockerLogs
```

Comment:
hier gibt es doch gar keine container oder übersehe ich da was?

Reply:
Scheinbar - siehe 
https://github.com/d-fine/Dataland/actions/runs/3336586206

Reply:
ja es funktioniert allgemein aber die frontend component tests werden ohne container ausgeführt, daher kann es auch keine logs geben und in dem referenzierten Run gibt es auch keine.

Reply:
habe ich da entfernt.